extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -238,6 +238,6 @@ class AmpStickyAd extends AMP.BaseElement {
   }
 }
 
-AMP.extension('amp-sticky-ad', AMP => {
+AMP.extension('amp-sticky-ad:0.1', AMP => {
```

Comment:
Yup, but to all other tests i think `amp-sticky-ad` and `amp-sticky-ad:0.1` makes no difference to them, it's just a symbol name that represent the id the of amp-sticky-ad installer and can be named to whatever thing.